pkg/apis/core/pods/helpers.go

```diff
@@ -25,19 +25,24 @@ import (
 )
 
 // ContainerVisitorWithPath is called with each container and the field.Path to that container
-type ContainerVisitorWithPath func(container *api.Container, path *field.Path)
+type ContainerVisitorWithPath func(container *api.Container, path *field.Path) bool
 
 // VisitContainersWithPath invokes the visitor function with a pointer to the spec
 // of every container in the given pod spec and the field.Path to that container.
```

Comment:
Describe how short circuiting works (can be copied from `pkg/api/v1/pod/util.go`)